src/sass/shared/components/footer.scss

```diff
@@ -93,6 +93,11 @@
           height: $size-footer-social-icon;
           border-radius: 50%;
           background-color: $color-footer-social-icon-bg;
+          transition: background-color 0.4s ease-in;
```

Comment:
In order to homogenize the transitions in the whole page, it should be used
```
@include transition(background-color);
```
That uses the global `$transition-main-time` too.